src/components/card/card.jsx

```diff
@@ -27,6 +27,31 @@ var Card = React.createClass({
     imageEl.src = this.props.thumbnail || Card.DEFAULT_THUMBNAIL;
   },
   render: function () {
+    function tagifyDescription (description) {
+      var words = description.split(' ');
+
+      if (words.length) {
+        words.forEach((word, index) => {
+          if (word.length)
+            if (words[index].match(/#([A-Za-z\d]+)/g)) {
+              words[index] = React.createElement(Link, {key: index, href: `/pages/tag-list`, url: `/tags/${word.slice(1)}`}, word);
+            } else {
+              words[index] = React.createElement(`span`, {key: index}, word);
+            }
+        });
+      }
+
+      // Add back the spaces between words
+      var interpolatedWords = [];
+
+      words.forEach((word, index) => {
+        interpolatedWords.push(word);
+        interpolatedWords.push(React.createElement(`span`, {key: `spacer-${index}`}, ` `));
+      });
+
+      return interpolatedWords.slice(0, -1); // Remove the trailing space
+    }
+
```

Comment:
I feel like there's probably a slightly better way to do this (split at hashtags instead of spaces, maybe? `string.split` can accept a regex...) but it does the job and I can't imagine it really affecting performance